Support/Scripts/ds2-gdb-wrapper.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
Yeah that's for systems where bash isn't in `/bin` which I think is usually
only BSDs. Doesn't matter too much in our case but it's still good to do it
that way.

On Thu, Jun 9, 2016 at 4:56 PM, kokarez <notifications@github.com> wrote:

> In Support/Scripts/ds2-gdb-wrapper.sh
> <https://github.com/facebook/ds2/pull/356#discussion_r66542412>:
>
> > @@ -0,0 +1,16 @@
> > +#! /bin/bash
>
> Also I think this is better or I think match other script.
>
>    - #!/usr/bin/env bash
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/356/files/556ef336524b1f05211a10186222fd68525c5c71..d047a9007eb41771f1c3099bdf3123c65041c990#r66542412>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbBRcaKx4eO-qHHWtkHxgDa0k_PIpks5qKKg4gaJpZM4InmxB>
> .
>



-- 
Stephane Sezer